resources/main.mjs

```diff
@@ -266,48 +269,19 @@ class MainBenchmarkClient {
     }
 
     _getFormattedCSVResult() {
-        let tests = [];
         // The CSV format is similar to the details view table. Each measurement is a row with
         // the name and N columns with the measurement for each iteration:
         // ```
         // Measurement,#1,...,#N
         // TodoMVC-JavaScript-ES5/Total,num,...,num
```

Comment:
I did a quick check with this applied and it drops the "/Total" from the subtest total string. I've been using that string to make analysis easy - is it possible to keep that?

Reply:
I'm a bit torn between that. I'd prefer having it all consistent between the metrics view and the exported data (minus the backwards-compatible JSON).
I'd be tempted to add `/Total` to any aggregated metric.

 - `TodoMVC-React-Redux` => `TodoMVC-React-Redux/Total`
 - `TodoMVC-React-Redux/Adding100Items` => `TodoMVC-React-Redux/Adding100Items/Total`
 
 WDYT?

Reply:
Sure, I can some up with another heuristic for splitting (ends in "/Total" and has a single "/" or something). Do you prefer your suggested approach here (/Total for any aggregate) or something else?